mindbender/maya/pythonpath/userSetup.py

```diff
@@ -2,6 +2,50 @@
 
 from maya import cmds
 
+import os
+
+
+def mayafpsconverter(Sfps):
```

Comment:
but in the bat files I wanna enter the string like:
> set MINDBENDER_FPS=25

and that will then be returned by 
> def mayafpsconverter(fps):

as 
> "pal"